src/main/kotlin/com/msg/gcms/domain/club/exception/OperationPlanNotFoundException.kt

```diff
@@ -0,0 +1,7 @@
+package com.msg.gcms.domain.club.exception
+
+import com.msg.gcms.global.exception.ErrorCode
+import com.msg.gcms.global.exception.exceptions.BasicException
+
+class OperationPlanNotFoundException: BasicException(ErrorCode.OPERATION_PLAN_NOT_FOUND) {
```

Comment:
msg 백엔드 코드 컨벤션에 상속에선 콜론앞에 공백추가 한다고 나와있어서
```suggestion
class OperationPlanNotFoundException : BasicException(ErrorCode.OPERATION_PLAN_NOT_FOUND) {
```
이런식으로 수정하면 좋을거 같아요.
<img width="769" alt="스크린샷 2023-06-16 오전 9 46 10" src="https://github.com/GSM-MSG/GCMS-BackEnd/assets/80161826/61ef7616-bb79-4f33-934f-e895b7859993">

Reply:
https://github.com/GSM-MSG/GCMS-BackEnd/pull/257/commits/39eea1f14f12853570a002434f9337c0101482e4
네.. 요건 몰랐네요 아래리뷰까지 해서 수정했습니다